internal/controller/numaresourcesscheduler_controller.go

```diff
@@ -361,7 +358,13 @@ func platformNormalize(spec *nropv1.NUMAResourcesSchedulerSpec, platInfo Platfor
 		return
 	}
 
-	parsedVersion, _ := platform.ParseVersion(activePodsResourcesSupportSince)
+	leastSupported, err := activepodresources.GetLeastSupportedByBuildType(platInfo.Version)
```

Comment:
ideally this should be computed once at startup: the platform should not change while the operator is running (but can change across operator restarts obviously).

Reply:
good point, I'll see where this goes better